api/Element.json

```diff
@@ -6877,16 +6945,22 @@
             ],
             "chrome_android": [
               {
-                "version_added": true
+                "version_added": "37"
               },
               {
                 "version_added": true,
                 "prefix": "webkit"
               }
             ],
-            "edge": {
-              "version_added": "13"
-            },
+            "edge": [
+              {
+                "version_added": "13"
+              },
+              {
+                "version_added": "79",
```

Comment:
Are you sure about this? If tests `webkitRequestPointerLock` are being used to infer this, can the other prefix support statements also be updated?

Reply:
This one was a manual update, since the collector doesn't have prefix support enabled.

Reply:
Ah, thanks!